locales/base/translation.json

```diff
@@ -1965,5 +1965,12 @@
       "title": "You need more {{feeTokenSymbol}} for gas",
       "description": "Add {{feeTokenSymbol}} to cover your gas fees or adjust the amount you are sending"
     }
+  },
+  "transactionFeed": {
```

Comment:
This name invites to put everything related to txs feed here, not only what is related to approvals. Is that the intent?

Reply:
yes that was my intention! transaction feed is really old and i notice that all the keys are on the top level and not very descriptive...so decided not to contribute to this pattern